cmd/ipfs-cluster-ctl/graph_test.go

```diff
@@ -172,19 +162,10 @@ C2 -> I2
 /* The swarm peer connections among ipfs daemons in the cluster */
 I1 -> I0
 I1 -> I2
-I1 -> I3
```

Comment:
This is fishy, can you look into it @kishansagathiya ?

Reply:
graph tests are failing because colours and colorschemes in tests are different than what we actually use. I am fixing that.

Reply:
What's fishy?

Latest build is passing.

I went through a bunch of previous (5-6) builds. I did not find anything graph related except one which was failing because of some tls related error.

Reply:
I'm talking about having to remove some edges for the tests to pass. Is the graph not producing some edges that it should in the allIpfs mode?

Reply:
It shouldn't produce those edge. ipfs links that we get from connect graph are swarmpeers, but in graph we don't show them all. 
We only show links among ipfs nodes which are linked to cluster peers.
So, removing those edges from `allIpfs` was the appropriate thing to do.

Reply:
`makeDot` takes an `allIpfs` parameter that enables/disabled (or should enable/disable) showing edges from all ipfs peer in the swarm. This test is testing that links exist for ipfs nodes not linked to cluster peers.

Reply:
Therefore this must be broken as right now both test produce the same output but the TestIpfsAllGraph has more ipfs links in the input.

Reply:
Ok. I see the problem. I will raise a PR against this.